make/asl_cflags.mk

```diff
@@ -19,17 +19,26 @@
 
 ASL_CFLAGS :=
 
-ASL_GDB := -ggdb
-ASL_CFLAGS += $(ASL_GDB)
+# Basic flags
+ASL_CFLAGS += -Os -std=gnu99
 
+#
+# Only specify ASL_DEBUG on devel builds. Reason being that when
+# we release RC builds to the field, we want them to be as close
+# to the official build as possible without being labeled as official.
+# That means no checks that would occur when DEBUG is active.
+#
+# STIEG: BUG: -DUSE_FULL_ASSERT - Not working on MK2.
```

Comment:
Blowing up.  We are using some improper values during init and the asserts are catching them since we never used them before now.  But I really wanted to get this out the door so i did this.